### ISSUE TITLE
Big Brother UK 2023

### ISSUE THREAD

**Issue by 10pmStalker** [Stremio/stremio-bugs#590]
It seems IMDB changed the way it's listed, they have it set up as a new series instead of a new season. The first 10 or so episodes worked fine before IMDB switched things up.

**Comment by 10pmStalker** [comment]
Just checked. Big brother is still unavailable

________________________________

**Comment by TRtomasz** [comment]
Uh, sorry about this will check again.

**Comment by 10pmStalker** [comment]
Any idea what the issue could be?

________________________________

**Comment by freezenification** [comment]
This still isn't fixed as far as I can tell?

**Comment by wedebe** [comment]
Still an issue for BBUK24; in the process of trying to find a solution or workaround, I came across a link on Reddit to 
https://videobrother.net/sub_pages/archives_uk

Unfortunately, last year's series is no longer available there, BUT... this archive hosts this year's after-show live feeds as well as the nightly and Late & Live shows!

**Comment by freezenification** [comment]
> Still an issue for BBUK24; in the process of trying to find a solution or workaround, I came across a link on Reddit to https://videobrother.net/sub_pages/archives_uk
> 
> Unfortunately, last year's series is no longer available there, BUT... this archive hosts this year's after-show live feeds as well as the nightly and Late & Live shows!

There's no way to watch Videobrother via Stremio though right? Unfortunately I don't watch stuff on my computer so using that website on a Fire Stick is a bit finicky.

**Comment by 10pmStalker** [comment]
> > Still an issue for BBUK24; in the process of trying to find a solution or workaround, I came across a link on Reddit to https://videobrother.net/sub_pages/archives_uk
> > Unfortunately, last year's series is no longer available there, BUT... this archive hosts this year's after-show live feeds as well as the nightly and Late & Live shows!
> 
> There's no way to watch Videobrother via Stremio though right? Unfortunately I don't watch stuff on my computer so using that website on a Fire Stick is a bit finicky.

Yeah I've come across that link, I'm in the same boat as you. It's Stremio or nothing right now. You can manually add each episode to Stremio with the magnet link but I can't even find this season on a public torrent site.
